### PR TITLE
Remove unused key types sr25 and ed25

### PR DESCRIPTION
**Pull request by JoshOrndorff**
The crypto `KeyTypeId`s are given four-character identifiers specific to their application such as `babe`, `gran`, `acco`, etc. This PR removes unused general purpose types `sr25` and `ed25`.

### CODE REVIEW

**Comment by bkchr** [comment]
DOes not compile and as you see, it is used in some test code. So, we should move the `KeyType` and the app crypto declaration into testing.

**Comment by bkchr** [comment]
Could you please move the both key types to `primitives/testing.rs` and the `app_crypto!` into `sr-primitives/testing.rs`?

File: core/application-crypto/src/ed25519.rs [hunk 1]
```diff
@@ -53,7 +53,14 @@ impl RuntimePublic for Public {
...
 mod tests {
 	use sr_primitives::{generic::BlockId, traits::ProvideRuntimeApi};
-	use primitives::{testing::KeyStore, crypto::Pair, traits::BareCryptoStore as _};
+	use primitives::{
+				testing::{
+						KeyStore,
+						ED25519,
+				},
+				crypto::Pair,
+				traits::BareCryptoStore as _
```
Comment (gavofyork) [inline core/application-crypto/src/ed25519.rs]:
```suggestion
				traits::BareCryptoStore as _,
```

File: core/application-crypto/src/sr25519.rs [hunk 2]
```diff
@@ -53,7 +53,14 @@ impl RuntimePublic for Public {
...
 mod tests {
 	use sr_primitives::{generic::BlockId, traits::ProvideRuntimeApi};
-	use primitives::{testing::KeyStore, crypto::Pair, traits::BareCryptoStore as _};
+	use primitives::{
+				testing::{
+						KeyStore,
+						SR25519,
+				},
+				crypto::Pair,
+				traits::BareCryptoStore as _
```
Comment (gavofyork) [inline core/application-crypto/src/sr25519.rs]:
```suggestion
				traits::BareCryptoStore as _,
```

**Pull request merged**